addons/website_slides/views/slide_slide_views.xml

```diff
@@ -92,6 +92,16 @@
                                 <label for="index_content"/>
                                 <field name="index_content"/>
                             </page>
+                            <page string="External Links">
+                                <group>
+                                    <field name="link_ids" widget="one2many" nolabel="1">
+                                         <tree editable="top">
+                                            <field name="name"/>
+                                            <field name="link"/>
```

Comment:
Looks like there is a "url" widget, not sure what it does or if it should be used here but it's maybe worth checking.